wiki/Introduction-to-Creating-a-Job.md

```diff
@@ -0,0 +1,1818 @@
+            - "{{Param.SceneFile}}"
+            - "{{Param.FramesDirectory}}"
+            - "{{Task.Param.RangeStart}}..{{Task.Param.RangeEnd}}"
+          timeout: 2400
+      embeddedFiles:
+        - name: Render
+          type: TEXT
+          filename: render.sh
+          runnable: true
+          data: |
+            #!/bin/bash
+
+            # Return an error code if any command in the script fails.
+            set -eou pipefail
+
+            TMPFILE=$(mktemp)
+            cat > "$TMPFILE" << EOF
+            import bpy
+            for s in bpy.data.scenes:
+              s.render.resolution_x = 480
+              s.render.resolution_y = 270
+            bpy.context.scene.cycles.samples = 100
+            EOF
+
+            SCENE="$1"
+            OUTDIR="$2"
+            FRAME="$3"
+
+            blender --background "$SCENE" \
+              --python "$TMPFILE" \
+              --render-output "$OUTDIR"/frame-### \
+              --render-format PNG --use-extension 1 \
+              --render-frame "${FRAME}"
+
+            rm -f $TMPFILE
+
+  - name: EncodeVideo
+    dependencies:
+      - dependsOn: BlenderRender
+    script:
+      actions:
+        onRun:
+          command: "{{Task.File.Encode}}"
+          args:
+            - "{{Param.FramesDirectory}}"
+            - "{{Param.AnimationFile}}"
+            - "{{Param.FrameStart}}"
+          timeout: 60
+      embeddedFiles:
+        - name: Encode
+          type: TEXT
+          runnable: true
+          filename: encode.sh
+          data: |
+            #!/bin/bash
+
+            set -eou pipefail
+
+            INPUT_DIR="$1"
+            OUTPUT_FILENAME="$2"
+            START_FRAME="$3"
+
+            ffmpeg -y -r 10 -start_number "$START_FRAME" -i "$INPUT_DIR"/frame-%03d.png -pix_fmt yuv420p \
+                -vf "scale=in_color_matrix=bt709:out_color_matrix=bt709" \
+                -frames:v 300 -c:v libx264 -preset fast \
+                -color_range tv -colorspace bt709 -color_primaries bt709 -color_trc iec61966-2-1 \
+                -movflags faststart "$OUTPUT_FILENAME"
+```
+
+##### 2.2.4.2.2 As STRING Type
+
+This option is only suitable for cases where you will be writing or programmatically generating a Job Template for a specific input
+because the `FrameStart` and `FrameEnd` values need to be known when defining the set of values for the Task parameter. To create the 
+parameter space, decide how many frames you want in each Task (we picked 40 in this case to cut down on typing) and then write out
+the frame expressions that Blender requires as the values of the Task parameter
+
+```yaml
+parameterSpace:
+      taskParameterDefinitions:
+        - name: Frames
+          type: STRING
+          range:
+          - "1..40"
+          - "41..80"
+          - "81..120"
+... and so on until
+          - "361-380"
+...
+```
+
+Also remove the definitions of the `FrameStart` and `FrameEnd` Job Parameters from the template because they are no longer used,
```

Comment:
I don't think we need the rest of this example to be fully demonstrated. We can list the caveats and gotchas and move on.